lychee-bin/src/commands/check.rs

```diff
@@ -261,9 +272,20 @@ async fn handle(
     //   benefit.
     // - Skip caching unsupported URLs as they might be supported in a
     //   future run.
-    // - Skip caching excluded links; they might not be excluded in the next run
+    // - Skip caching excluded links; they might not be excluded in the next run.
+    // - Skip caching links for which the status code has been explicitly excluded from the cache.
     let status = response.status();
-    if uri.is_file() || status.is_excluded() || status.is_unsupported() || status.is_unknown() {
+    let status_code = match status.code() {
+        None => 0,
+        Some(code) => code.as_u16(),
+    };
+
+    if uri.is_file()
+        || status.is_excluded()
+        || status.is_unsupported()
+        || status.is_unknown()
+        || cache_exclude_status.contains(&status_code)
```

Comment:
This is not tested, but I didn't see any tests for the `handle()` method.

Reply:
Thanks for thinking about testing.
Can we extract that part into its own function and add some unit tests?
I think that would be the easiest.

Reply:
Something like `fn skip_cache(uri, status) -> bool;`

Reply:
I've gone with `ignore_cache`, but I can use `skip` instead.

Reply:
Looks good.